modules/libraries/src/views/full/Dropdown.tsx

```diff
@@ -0,0 +1,61 @@
+import { Button, Classes, IconName, MenuItem } from '@blueprintjs/core'
+import { Suggest } from '@blueprintjs/select'
+import React, { FC, useState } from 'react'
+
+export interface Option {
+  name: string
+  version: string
+  icon?: IconName | JSX.Element
+}
+
+const itemRenderer = (option, { modifiers, handleClick }) => {
+  if (!modifiers.matchesPredicate) {
+    return null
+  }
+
+  return (
+    <MenuItem
+      className={Classes.SMALL}
+      active={modifiers.active}
+      disabled={modifiers.disabled}
+      key={option.name || option}
+      onClick={handleClick}
+      text={
+        <div style={{ display: 'flex', justifyContent: 'space-between' }}>
+          <div>
+            <strong>{option.name}</strong> <br></br>
+            {(option.description || '').slice(0, 150)}
+          </div>
+          <div>{option.version}</div>
+        </div>
+      }
+      icon={option.icon}
+    />
+  )
+}
+
+const Dropdown: FC<any> = props => {
```

Comment:
That naming is to me, way too generic. Can you name it as something a bit more precise like `LibrariesDropdown` or dependencyDropdown. At frist I thought you were re-writing shared dropdown again ...